spec/rubocop/cop/lint/unescaped_bracket_in_regexp_spec.rb

```diff
@@ -229,6 +229,22 @@
           RUBY
         end
       end
+
+      context 'invalid regular expressions' do
+        it 'does not register an offense' do
+          expect_no_offenses(<<~RUBY)
+            Regexp.#{method}("*")
+          RUBY
+
+          expect_no_offenses(<<~RUBY)
+            Regexp.#{method}("+")
+          RUBY
+
+          expect_no_offenses(<<~RUBY)
+            Regexp.#{method}("{42}")
+          RUBY
+        end
```

Comment:
Can you write `it 'does not register an offense ...' do` for each `expect_no_offenses`?

Reply:
Done.